src/utils/webAppUtil.ts

```diff
@@ -0,0 +1,90 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ApiContract, BackendContract, BackendCredentialsContract } from "azure-arm-apimanagement/lib/models";
+import WebSiteManagementClient from "azure-arm-website";
+import { Site, WebAppCollection } from "azure-arm-website/lib/models";
+import xml = require("xml");
+import { ApisTreeItem } from "../explorer/ApisTreeItem";
+import { ApiTreeItem } from "../explorer/ApiTreeItem";
+import { ext } from "../extensionVariables";
+import { azureClientUtil } from "./azureClientUtil";
+import { nonNullOrEmptyValue } from "./nonNull";
+
+export namespace webAppUtil {
```

Comment:
these can be in importwebapp and importfunctionapp can import these functions. dont need a util for it.